src/Platform.elm

```diff
@@ -63,6 +63,85 @@ Initializing a headless program from JavaScript looks like this:
 ```javascript
 var app = Elm.MyThing.worker();
 ```
+
+### init
+
+The first argument, `init` is a [Tuple](Tuple) that lets you specify how your program should start.
+Its first value is the initial state of your program's model.
+The second value is an initial command. Often that command is [Cmd.none](Platform-Cmd#none).
+
+### update
+
+The second argument, `update` is a function that takes these two arguments:
+
+1. A message from the outside world, that has the information your code will act upon.
+Usually this is a `Msg` type that you specify, (a constructor), which takes a record type
+matching the data coming from an incoming port. Unpack that data with `case` ... `of`.
+
+2. The current state of your program's model, as maintained by the Elm runtime.
+
+The `update` function must return a [Tuple](Tuple)
+containing the modified form of your program's model, and a command.
```

Comment:
This does not appear to be true. I've been debugging a program here - https://github.com/Pilatch/elm-port-program-examples/blob/master/greeter/Greet.elm - and can't find any evidence to show that update is getting called multiple times when I supply a non-none command in the second half of the tuple that's returned from update.

Here I have the result of debugging the result of a Cmd.map:

    { type = "map", tagger = <function>, tree = { type = "leaf", home = "greet", value = "jomomma" } }

Here is the result of debugging Cmd.none:

    { type = "node", branches = [] }

They are significantly different, yet in both cases the update function is not run again, with the supplied message coming from Cmd.map.